prow/plugins.yaml

```diff
@@ -51,8 +51,11 @@ approve:
   - kubernetes/sig-release
   - kubernetes/steering
   - kubernetes/test-infra
-  - kubernetes/website
   implicit_self_approve: true
+- repos:
+  - kubernetes/website
+  issue_required: false
+  implicit_self_approve: false
```

Comment:
This also defaults to false.
It should be sufficient to just delete the reference to `kubernetes/website` in the `approve.repos` config. The `approve` plugin is still enabled for the repo lower in this file, but now that you are switching to all default config for the plugin you don't need to specify its configuration.